app/build.gradle

```diff
@@ -37,7 +37,9 @@ android {
     execution 'ANDROIDX_TEST_ORCHESTRATOR'
     unitTests {
       includeAndroidResources = true
+
```

Comment:
Remove this extra line.

Reply:
resolved

Reply:
You have marked this resolved but this has not been resolved.

Reply:
the changes were not pushed successfully